src/components/BoardUsers/BoardUser.scss

```diff
@@ -19,21 +19,20 @@
 }
 
 .rest-users {
-  color: $color-backlog-blue;
-  border: 2px solid $color-backlog-blue;
-  height: $icon--extralarge;
-  width: $icon--extralarge;
+  background-color: #333948;
+  color: $color-white;
+  height: 36px;
+  width: 36px;
```

Comment:
I guess 32px is too small? (in case it's not `$icon--large: 32px;`)

Reply:
I think it's too small. Looks strange in contrast to the larger avatars